Model/StringFormatter.php

```diff
@@ -0,0 +1,171 @@
+<?php
+declare(strict_types=1);
+
+namespace Bazaarvoice\Connector\Model;
+
+use Bazaarvoice\Connector\Api\ConfigProviderInterface;
+use Bazaarvoice\Connector\Api\StringFormatterInterface;
+
+/**
+ * Class StringFormatter
+ *
+ * @package Bazaarvoice\Connector\Model
+ */
+class StringFormatter implements StringFormatterInterface
+{
+    /**
+     * @var ConfigProviderInterface
+     */
+    private $configProvider;
+
+    /**
+     * StringFormatter constructor.
+     *
+     * @param ConfigProviderInterface $configProvider
+     */
+    public function __construct(ConfigProviderInterface $configProvider)
+    {
+        $this->configProvider = $configProvider;
+    }
+
+    /**
+     * This unique ID can only contain alphanumeric characters (letters and numbers
+     * only) and also the asterisk, hyphen, and underscore characters. If your
+     * product IDs contain invalid characters, simply replace them with an alternate
+     * character like an underscore. This will only be used in the feed and not for
+     * any customer facing purpose.
+     *
+     * @static
```

Comment:
Why is this declared as static, but some of the other functions like jsonEncode are not?

Reply:
Removed @static. Not sure where that came from.